src/Term/LabelsProvider.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Wikibase\DataModel\Term;
+
+/**
+ * @since 3.1
+ *
+ * @licence GNU GPL v2+
+ * @author Bene* < benestar.wikimedia@gmail.com >
+ */
+interface LabelsProvider {
+
+	/**
+	 * @return TermList
```

Comment:
Same as in https://github.com/wmde/WikibaseDataModel/pull/513#discussion_r33853410.